### PR TITLE
New Port Super Cakeboy

### PR DESCRIPTION
**Pull request by Fraxinus88**
# New Port for Super Cakeboy

## Game Information
- **Title**: Super Cakeboy
- **URL**: https://store.steampowered.com/app/2319580/Super_Cakeboy/

### CFW Tests:
[X] AmberELEC
[X] ArkOS
[X] JELOS/ROCKNIX
[X] MuOS

Architecture
[X] armHF
[X] Panfrost (Mainline)

Resolution Tests
Test all major resolutions:
[X] 480x320
[X] 640x480
[X] 720x720 (RGB30)
[X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/supercakeboy/Super Cakeboy.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
```
Comment (Cebion) [inline ports/supercakeboy/Super Cakeboy.sh:3]: Old header
https://portmaster.games/packaging.html

File: ports/supercakeboy/gameinfo.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./Super Cakeboy</path>
```
Comment (Cebion) [inline ports/supercakeboy/gameinfo.xml:4]: gameinfo.xml path needs to point to the .sh

File: ports/supercakeboy/screenshot.png
Comment (Cebion) [inline ports/supercakeboy/screenshot.png:1]: Screenshot needs to be 640x480 and as be as small as possible in filesize.
Use https://squoosh.app/ for example to reduce images filesizes

**Review by Cebion — CHANGES_REQUESTED** [review]

**Review by Cebion — DISMISSED** [review]

File: ports/supercakeboy/gameinfo.xml (hunk 2 above)
Comment (kloptops) [reply ports/supercakeboy/gameinfo.xml:4]: This still needs fixing.

**Review by kloptops — CHANGES_REQUESTED** [review]
Just need to fix the gameinfo.xml 😁

**Comment by Fraxinus88** [comment]
Whoopsies and done!

**Pull request merged**